testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -231,12 +231,13 @@ public void afterAbstractKeycloakTest() {
                 }
             }
             testContext.getCleanups().clear();
-        }
 
-        postAfterAbstractKeycloak();
 
-        // Remove all browsers from queue
-        DroneUtils.resetQueue();
+            postAfterAbstractKeycloak();
```

Comment:
I am not sure about this change and why is it needed? I think the "postAfterAbstractKeycloak()" and "DroneUtils.resetQueue()" need to be called also when "importAfterEachMethod" is true? Which is changed here.